src/Swarm/Game/Step.hs

```diff
@@ -1346,7 +1346,7 @@ execConst c vs s k = do
       [VBool b] -> return $ Out (VBool (not b)) s k
       _ -> badConst
     Neg -> case vs of
-      [VInt n] -> return $ Out (VInt (-n)) s k
+      [VInt n] -> return $ Out (VInt (- n)) s k
```

Comment:
OMG, I really have to find out which version of Fourmolu Restyled uses :expressionless: